pkg/artifactory/resource_artifactory_remote_repository.go

```diff
@@ -261,8 +263,8 @@ func resourceArtifactoryRemoteRepository() *schema.Resource {
 			"content_synchronisation": {
 				Type:     schema.TypeList,
 				Optional: true,
+				Computed: true,
 				MaxItems: 1,
-				MinItems: 1,
```

Comment:
When Optional is set to `true`, MinItems is ignored

Reference: https://github.com/hashicorp/terraform-plugin-sdk/blob/b9a5ed2f57da2dcdb700d4295f91badd0b4793f4/helper/schema/schema.go#L165-L166